Fix single consignment

andresa-razoyo opened the pull request.

andresa-razoyo committed:
Fix single consignment

edias-razoyo commented on public/js/bigcommerce.js:
```diff
@@ -1160,26 +1165,43 @@ async function preventSubmissionOnShipping() {
+        element.addEventListener('click', async (event) => {
+            event.preventDefault();
             if (FFLConfigs.preventSubmition && !FFLConfigs.selectedDealer) {
                 event.preventDefault();
```
this line could be removed since you added it on the function beginning

andresa-razoyo replied:
Not sure if this is the same as you saw before, but this can't be removed

andresa-razoyo committed:
Verify if shipping address can have ammo send to it

Verifies which consignment has ammo (single or multishipping checkout). Displays message to the user, and send him back to the state selection.

edias-razoyo commented on public/js/bigcommerce.js:
```diff
@@ -535,6 +548,21 @@ async function deleteAllConsignments() {
+        await new Promise(resolve => setTimeout(resolve, 500));
+
+        // We need to get them again to update the state
+        const updatedData = await fetchGraphQLData(`
```
It could be moved to graphqlPayloads variable

andresa-razoyo committed:
Add queries to graphqlPayloads

andresa-razoyo committed:
Turn shippingConsignmentsDelete into a function
